client/src/reducers/lessonReducer.ts

```diff
@@ -25,6 +26,8 @@ export interface Lesson {
 
 export type LessonState = {[id: number]: Lesson};
 
+export const parts_of_speech = ['noun', 'verb', 'adjective', 'adverb'];
```

Comment:
This is not a large enough list.
You should consolidate with: https://github.com/teamdotjs/v2/blob/master/client/src/components/Lesson/WordCreator/WordFormSelector.tsx#L19